app_backend/app/auth_handler.py

```diff
@@ -14,9 +14,10 @@ def token_response(token: str):
         "access_token": token
     }
 
-def signJWT(user_id: str) -> Dict[str, str]:
+def signJWT(user_id: str, email: str) -> Dict[str, str]:
```

Comment:
I think you don't have to include email in JWT token, but it's also fine if you do.

Reply:
I agree -- I think UUID is the more important one. Currently, the JWT only stores the email, so this comment is a little misleading. I am moving the email to a separate parameter and adding UUID as the new first parameter as we care about that one more. Should we get rid of email in it's entirety?